easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -558,6 +560,15 @@ def install_step(self):
         if pythonpath is not None:
             env.setvar('PYTHONPATH', pythonpath, verbose=False)
 
+        for glob_pattern in self.cfg.get('fix_python_shebang_for', []):
+            paths = glob.glob(os.path.join(self.installdir, glob_pattern))
+            self.log.info("Fixing shebang to '%s' for files that match '%s': %s", PYTHON_SHEBANG, glob_pattern, paths)
+            for path in paths:
+                apply_regex_substitutions(path, [(r'^#!.*python$', PYTHON_SHEBANG)])
```

Comment:
Need to be python[0-9.]*$ to cover all cases.

Reply:
For python 3.7.2 it puts
#!/hpc2n/eb/software/Compiler/GCCcore/8.2.0/Python/3.7.2/bin/python3.7
in the file it created.

Reply:
Good point, thanks.

Reply:
Fixed